openslides/agenda/static/js/agenda/base.js

```diff
@@ -52,11 +52,11 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
                         // in the DS store.
                         title = this.title;
                     }
-                    if (this.getContentResource().verboseName) {
-                        title = gettextCatalog.getString(this.getContentResource().verboseName) + ' ' + title;
+                    if (this.getContentResource().agendaSupplement) {
+                        title = gettextCatalog.getString(this.getContentResource().agendaSupplement) + ' ' + title;
                     }
                     if (this.item_number) {
-                        title = this.item_number + ' ' + title;
+                        title = this.item_number + ' · ' + title;
```

Comment:
This is not a simple point ("."). This is a middle dot ("&middot;") which is used as separator between item_number and title. The symbol should separate the automated number from the custom title. And the middle dot works for all forms of item_number, see e.g. "TOP 1 &middot; Title" or "a) &middot; Title"
So I won't fix it here.